src/uthash.h

```diff
@@ -423,7 +480,7 @@ do {
  */
 #ifdef HASH_DEBUG
 #define HASH_OOPS(...) do { fprintf(stderr,__VA_ARGS__); exit(-1); } while (0)
-#define HASH_FSCK(hh,head)                                                       \
+#define HASH_FSCK(hh,head,fun)                                                   \
```

Comment:
This seems like a good idea but orthogonal to the error-handling stuff. If you rename `fun` to `where` throughout, and then submit this as a separate patch, I'll accept it.